Apps/Medication/src/Services/RestMedicationStatementService.cs

```diff
@@ -94,6 +94,11 @@ private async Task<HNMessage<List<MedicationStatement>>> RetrieveMedicationState
                 string jwtString = this.httpContextAccessor.HttpContext.Request.Headers["Authorization"][0];
                 string phn = await this.patientDelegate.GetPatientPHNAsync(hdid, jwtString).ConfigureAwait(true);
 
+                if (string.IsNullOrEmpty(phn))
+                {
+                    return new HNMessage<List<MedicationStatement>>() { Result = Common.Constants.ResultType.Error, ResultMessage = "PHN could not be retrieved" };
```

Comment:
I created an Error Messages constants class that should be used for error messages like this.

Reply:
good call. I've added that one to the constants and replaced it here.